aws-s3/s3.mli

```diff
@@ -222,9 +222,20 @@ module Make(Io : Types.Io) : sig
         the etag return is _NOT_ the md5 *)
     val complete : (t -> unit -> etag result) command
 
+(** Complete a multipart upload with explicit parameters.
+    This is useful for stateless workflows where upload metadata is stored in a database.
+    The returned string is an opaque identifier used as etag.
+    the etag return is _NOT_ the md5 *)
```

Comment:
Could the `Multipart.t` not be used here as it is in `complete` so `complete` and `complete_stateless` have identical signatures?